docs/user-event/api-pointer.mdx

```diff
@@ -0,0 +1,120 @@
+We don't try to determine if the pointer action you describe is possible at that
+position in your layout.
+
+## Pointer action
+
+There are to types of actions: press and move.
+
+### <a name="press" href="#"/>Pressing a button or touching the screen
+
+A pointer action is a press action if it defines a key to be pressed, to be
+released, or both.
+
+```js
+pointer({keys: '[MouseLeft]'})
+```
+
+You can declare multiple press actions (on the same position) at once which will
+be resolved to multiple actions internally. If you don't need to declare any
+other properties you can also just supply the `keys` string.
+
+```js
+pointer({keys: '[MouseLeft][MouseRight]'})
+// or
+pointer('[MouseLeft][MouseRight]')
+```
+
+For pressing a button without releasing it, the button name is suffixed with
+`>`.  
+For just releasing a previously pressed button, the tag is started with `/`.
+
+```js
+pointer('[MouseLeft>]') // press the left mouse button
+pointer('[/MouseLeft]') // release the left mouse button
+```
+
+Which button are available depends on the [`pointerMap`](options#pointermap).
+
+### <a name="move" href="#"/>Moving a pointer
+
+Every pointer action that is not a press action describes a pointer movement.
+
+You can declare which pointer is moved per `pointerName` property. This defaults
+to `mouse`.
+
+Note that the `mouse` pointer (`pointerId: 1`) is also the only pointer that
+always exists and has a position. A `touch` pointer only exists while the screen
+is touched and receives a new `pointerId` every time. For these we therefore use
+the "button" name from the press action as `pointerName`.
+
+```js
+pointer([
+  // touch the screen at element1
+  {keys: '[TouchA>]', target: element1},
+  // move the touch pointer to element2
+  {pointerName: 'TouchA', target: element2},
+  // release the touch pointer at the last position (element2)
+  {keys: '[/TouchA]'},
+])
+```
+
+## Pointer position
+
+### PointerTarget
+
+```ts
+interface PointerTarget {
+  target: Element
+  coords?: PointerCoords
+}
+```
+
+The `PointerTarget` props allows to describe the position of the pointer on the
+document.  
+The `coords` you provide are applied as-is to the resulting
+[`MouseEvent`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) and
+can be omitted.  
+The `target` should be the element receiving the pointer input in the browser.
+This is the topmost element that can receive pointer event at those coordinates.
+
+### SelectionTarget
+
+```ts
+interface SelectionTarget {
+  node?: Node
+  offset?: number
+}
+```
+
+Pointer actions can alter the selection in the document.  
+In the browser every pointer position corresponds with a DOM position. This is a
+DOM node and a DOM offset which usually translates to the character closest to
+the pointer position.  
+As all character in a no-layout environment are in the same layout position we
+assume a pointer position to be closest to the last descendant of the pointer
+`target`.
```

Comment:
I guess that it's related to the fact that JSDOM has no layout mechanism so all elements have the same position?

Reply:
Why should we add a comma here? :thinking: 

> As all character in a no-layout environment are in the same layout position

There is no layout. So every character has *effective* width zero and is at *effective* offset `0,0`. (While there might be properties applied that suggest otherwise but have no effect.)

Reply:
👍 I see thanks for the elaboration, the term "no-layout" wasn't clear to me.
As for the comma, it's to make it easier to read imho. It's also something that grammarly (a spelling tool) suggests.

Reply:
I'm no linguistic expert :shrug: 
If I understood some articles about this correctly, generally an introductory phrase should have a comma. But it can be omitted for prepositional phrases shorter than five words and should be omitted if there is no natural pause for building suspension and/or the following sentence could not stand on its own.
I'd judge `Every pointer position corresponds with a DOM position.` on its own as technically incorrect, as this paragraph just explains that this is not the case in the no-layout environment. Following this, I think it should be there, but again, I'm no expert on this. (Which might just have been highlighted by the fact that muscle-memory typed `export` instead of `expert` twice in this post :laughing: )

Reply:
Yea, me neither 😅
I just added it because I had a corresponding question that went with it.
Thanks again @ph-fritsche !